subtests/docker_cli/create/create_tmpfs.py

```diff
@@ -0,0 +1,38 @@
+"""
+Create container with --tmpfs option and inspect it
+"""
+
+from create import create_base
+from dockertest.output import DockerVersion
+from dockertest.dockercmd import DockerCmd
+from dockertest.output import mustpass
+
+
+class create_tmpfs(create_base):
+
+    def initialize(self):
+        super(create_tmpfs, self).initialize()
+        # Minimum docker version 1.10 is required for --tmpfs
+        DockerVersion().require_server("1.10")
+
+    def run_once(self):
+        super(create_tmpfs, self).run_once()
+
+    def postprocess(self):
+        # Inspect container for --tmpfs
+        subargs = ["--format", "'{{.HostConfig.Tmpfs}}'",
+                   self.sub_stuff['name']]
+        dkrcmd = DockerCmd(self, "inspect", subargs)
+        self.sub_stuff['dkrcmd'] = dkrcmd
+        self.sub_stuff['cmdresult'] = mustpass(dkrcmd.execute())
+        tmpfs_arg = self.config['tmpfs_arg']
+        with_tmpfs = self.sub_stuff['dkrcmd'].stdout.find(tmpfs_arg) != -1
+        self.failif(not with_tmpfs,
+                    "Container was not created with: --tmpfs %s"
+                    % tmpfs_arg)
+        # Change inspect paramerter to get CID
+        subargs = ["--format", "'{{.Id}}'", self.sub_stuff['name']]
+        dkrcmd = DockerCmd(self, "inspect", subargs)
+        self.sub_stuff['dkrcmd'] = dkrcmd
+        self.sub_stuff['cmdresult'] = mustpass(dkrcmd.execute())
```

Comment:
I think would be better to not re-use both the ``dkrcmd`` instance _and_ the execution result, pick one or neither.  In fact, there's a ``cmdresult`` built-in to the ``dkrcmd`` instance already, so you don't even really need to store it separately - similar to how ``create_base.postprocess()`` does it.